.dependabot/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 1
```

Comment:
@vexx32, your last commit had 67 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Basic Registry Provider Tests.Validate basic registry provider Cmdlets.Verify Get-ChildItem
```powershell
Expected exactly 2, but got 37.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Registry.Tests.ps1: line 68
68:             $items.Count | Should -BeExactly 2
```
Basic Registry Provider Tests.Validate basic registry provider Cmdlets.Verify Get-ChildItem can get subkey names
```powershell
Expected exactly 2, but got 37.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Registry.Tests.ps1: line 75
75:             $items.Count | Should -BeExactly 2
```
Basic Registry Provider Tests.Valdiate basic registry property Cmdlets.Verify New-ItemProperty
```powershell
Expected 99, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Registry.Tests.ps1: line 113
113:             $property.NewTestEntry | Should -Be 99
```
Basic Registry Provider Tests.Valdiate basic registry property Cmdlets.Verify Set-ItemProperty
```powershell
Expected 2, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Registry.Tests.ps1: line 120
120:             $property."$testPropertyName" | Should -Be 2
```
Basic Registry Provider Tests.Valdiate basic registry property Cmdlets.Verify Set-Item
```powershell
Provider operation stopped because the provider does not support this operation.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Registry.Tests.ps1: line 124
124:             Set-Item -Path $testKey -Value $defaultPropertyValue

```